### PR TITLE
Add label aggregation for innodb buffer metrics

### PR DESCRIPTION
**Pull request by SuperQ**
Add label aggregation for innodb_metrics buffer subsystem.
* Total can be ignored as it is an aggregation of data+misc+free
* Dirty pages are not aggregateable, keep as a separate metric.

### CODE REVIEW

**Comment by SuperQ** [comment]
@grypyrg This should help with https://github.com/prometheus/mysqld_exporter/issues/129.

**Comment by SuperQ** [comment]
Ping @grypyrg, is this what you're looking for?

**Comment by SuperQ** [comment]
Ping @grypyrg

File: collector/info_schema_innodb_metrics_test.go [hunk 1]
```diff
@@ -30,7 +30,10 @@ func TestScrapeInnodbMetrics(t *testing.T) {
 		AddRow("buffer_pool_size", "server", "value", "Server buffer pool size (all buffer pools) in bytes", 2).
 		AddRow("buffer_page_read_system_page", "buffer_page_io", "counter", "Number of System Pages read", 3).
 		AddRow("buffer_page_written_undo_log", "buffer_page_io", "counter", "Number of Undo Log Pages written", 4).
-		AddRow("NOPE", "buffer_page_io", "counter", "An invalid buffer_page_io metric", 5)
+		AddRow("buffer_pool_pages_dirty", "buffer", "gauge", "Number of Undo Log Pages written", 5).
```
Comment (grypyrg) [inline collector/info_schema_innodb_metrics_test.go]: Maybe `"Number of Undo Log Pages written"` needs to be changed in these rows?

**Comment by grypyrg** [comment]
Apart from the code comment, looks good to me.


 (sorry for delay in replying)

**Comment by SuperQ** [comment]
@grypyrg Fixed comment in tests.

**Pull request merged**